### ISSUE TITLE
pulp file access 

### ISSUE THREAD

**Issue by lfernand4** [pulp-platform/pulp-rt-examples#2]
I try to run the exemple "pulp-rt-examples/bridge/file/" but I don't understant how it work. 
The program is blocking here : 
![image](https://user-images.githubusercontent.com/36438648/44454788-bdd51480-a5fc-11e8-9607-f345c2dff754.png)

 Thank you in advance.

**Comment by haugoug** [comment]
This feature is for now only used when running on a real board.

**Comment by lfernand4** [comment]
It is possible to send data via a virtual uart ?

**Comment by haugoug** [comment]
Yes you can try this example: examples/pulp-rt-examples/periph/uart/send
The uart bit stream which is sent by the test is dumped to this file: build/pulp/tx_uart.log
Let me know if you have any issue with this test

**Comment by lfernand4** [comment]
I have the following error : 
make: *** [run] Segmentation fault (core dumped)
